bokeh/models/widgets/inputs.py

```diff
@@ -208,6 +208,12 @@ class Spinner(NumericInput):
 
     '''
 
+    def __init__(self, **kwargs):
+        if "value" in kwargs and "value_throttled" not in kwargs:
+            kwargs["value_throttled"] = kwargs["value"]
```

Comment:
Is this actually necessary? The main use case for `value_throttled` is to attach change handlers to, it seems OK to me if it starts out unset and raises a an unset value error if it is explicitly read.

Reply:
This was requested by @philippjfr, so that if `value` was configured then accessing `value_throttled` in a callback before any interactions happened, would not result in an exception. This is supposed to simplify design of callbacks.

Reply:
Well, maybe longer term we can add a mechanism to delegate initialization based on another property. I am sure you are also not happy to see initializers added to models.

Reply:
That's one option (that I actually considered). However, I also don't like `value_throttled` in general. I will be looking into replacing this with a more generic mechanism, that would work for any properties, without having to duplicate their definitions.

Reply:
TBH it does not seem worth much effort. Throttling is not generally applicable, it only makes sense on a handful of properties, and then in all those cases, implementing throttling requires custom code on the BokehJS side to adapt to whatever specific situation or widget is going on on the JS. The previous policy based approach was both more complicated and also limiting so I don't think going back down that road makes sense either. If you literally just mean some shortcut on the Python side to auto-generate an extra `_throttled` version of a property e.g if some `throttleable` flag is set, then sure, but that also seems of marginal value.

Reply:
Or maybe you mean some generic system to have some way to deal with back pressure. Thats fine (but a large undertaking) but also orthogonal to this kind of throttling. It's reasonable for a user to want to say "I only care about the end value of a completed UX event, ignore the middle" with or without back pressure